lib/Appium/TouchActions.pm

```diff
@@ -6,7 +6,7 @@ use Moo;
 has 'driver' => (
     is => 'ro',
     required => 1,
-    handles => [ qw/execute_script/ ]
+    handles => [ qw/prompt/ ]
```

Comment:
Sorry, I don't understand what this `handles` entry is for. I searched for `prompt` in TouchActions.pm and Appium.pm but didn't find any entries. Do you remember why it's here?